lib/queuePopulator/LogReader.js

```diff
@@ -379,20 +379,8 @@ class LogReader {
 
     async _processLogEntry(batchState, record, entry) {
         return Promise.all(this._extensions.map(ext => {
-            /**
-             * Delete entries don't have a value field.
-             * Value field normally contains the timestamp
-             * of the event, which is needed for the notification
-             * extension.
-             * That gets added here, we use the oplog event timestamp
-             */
-            if (entry.type === 'delete' &&
-                ext.constructor.name === 'NotificationQueuePopulator') {
-                entry.value = JSON.stringify({
-                    'last-modified': record.timestamp
-                });
-            // skipping entry for other extensions
-            } else if (entry.value === undefined) {
+            // skipping delete entry for all extensions
+            if (entry.value === undefined) {
                 return null;
             }
             return ext.filter({
```

Comment:
if we detect this 'update' is actually a 'delete', should we not update the record, so that downstream extensions receive a 'delete' as before?